Resources/Prototypes/DeltaV/Catalog/Fills/Boxes/syndicate.yml

```diff
@@ -0,0 +1,12 @@
+- type: entity
+  id: TacticalMaidOutfit
```

Comment:
This should probably have an ID indicating that it's a box of something instead of saying it's clothing

Reply:
I do kinda agree on that one, I just didn't want to make it TOO too clogged up as a name, will do